.semaphore/semaphore.yml

```diff
@@ -6,8 +6,8 @@ execution_time_limit:
 
 agent:
   machine:
-    type: e1-standard-2
-    os_image: ubuntu1804
+    type: e2-standard-2
```

Comment:
had to change the machine type too (semaphore doesn't offer ubuntu2204 on e1-standard-2 :shrug:)